manifests/init.pp

```diff
@@ -62,6 +66,7 @@
 class chocolatey (
   $choco_install_location         = $::chocolatey::params::install_location,
   $use_7zip                       = $::chocolatey::params::use_7zip,
+  $binary_7zip                    = $::chocolatey::params::binary_7zip,
```

Comment:
@ferventcoder, are you ok with this new param name?

Reply:
That doesn't self document what it is. Needs something like `7zip_download_url` to make it consistent with the other params.

Reply:
Certainly, tough I cannot start a var name with a number. What about:
`$download_url_7zip`

Reply:
Ouch. How about `seven_zip_download_url`? Although both would probably work, just weird workaround that folks may not immediately understand.